### PR TITLE
update to 0.13.40

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
Moved #8 forward

### CODE REVIEW

**Review by cocreature — DISMISSED** [review]
Thanks for fixing this so quickly!

**Comment by remyhaemmerle-da** [comment]
need more work

**Comment by stefanobaghino-da** [comment]
I have rephrased the original PR description to prevent GitHub to automatically close the ticket when this gets merged.

File: src/main/java/examples/pingpong/components/PingPongComponentsMain.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ private static void createInitialContracts(LedgerClient client, String sender, S
 
             // The wrapper object for commands with more details for the resulting transaction
             client.getCommandSubmissionClient().submit(
-                    String.format("Ping-%s-%d", sender, i),
+                    String.format("B-Ping-%s-%d", sender, i),
```
Comment (stefanobaghino-da) [inline src/main/java/examples/pingpong/components/PingPongComponentsMain.java]:
```suggestion
                    String.format("Ping-%s-%d", sender, i),
```
Looks like a leftover from some debugging session. :wink:

**Review by stefanobaghino-da — DISMISSED** [review]
Thanks!

**Comment by remyhaemmerle-da** [comment]
The following commands have been succesfully tested:

* ```mvn exec:java -Dexec.mainClass=examples.pingpong.reactive.PingPongReactiveMain "-Dexec.args=localhost 7600"```
* ``` mvn exec:java -Dexec.mainClass=examples.pingpong.grpc.PingPongGrpcMain "-Dexec.args=localhost 7600"```

**Review by gerolf-da — APPROVED** [review]

**Pull request merged**